fedora/templates/csv/file_dir_permissions.csv

```diff
@@ -2,10 +2,10 @@
 /etc,gshadow,0,0,0000
 /etc,passwd,0,0,0644
 /etc,group,0,0,0644
-/etc,cron.allow,,,0644,cron_allow
-/etc/httpd/conf.d,*,,,0640,httpd_server_conf_d_files
-/etc/httpd/conf,*,,,0640,httpd_server_conf_files
-/etc/ssh,*.pub,,,0644,sshd_pub_key
-/etc/ssh,*_key,,,0640,sshd_private_key
-/etc/httpd/conf.modules.d,*,,,0640,https_server_modules_files
+/etc,cron.allow,0,0,0644,cron_allow
+/etc/httpd/conf.d,^.*$,,,0640,httpd_server_conf_d_files
+/etc/httpd/conf,^.*$,,,0640,httpd_server_conf_files
+/etc/ssh,^.*.pub$,,,0644,sshd_pub_key
```

Comment:
I think he second dot in the regex should be escaped since it should mean literally a dot, not any character.